scripts/ci/in_container/check_environment.sh

```diff
@@ -122,12 +122,19 @@ function check_db_connection {
 function check_mysql_logs {
     MAX_CHECK=${1:=60}
     # Wait until mysql is ready!
-    MYSQL_CONTAINER=$(docker ps -qf "name=mysql")
+    MYSQL_CONTAINER=$(docker ps -q -f "name=mysql" -f "volume=/dev/urandom")
```

Comment:
It could use `docker ps -qf "name=docker-compose_mysql_1"`, but I'm not sure if other would contain the name `docker-compose_mysql_1*`

Reply:
Why don't we use docker-compose here? 
<img width="804" alt="Screenshot 2020-04-03 at 15 24 34" src="https://user-images.githubusercontent.com/12058428/78365324-3ebd0600-75bf-11ea-8652-c711edd99fb3.png">

Reply:
Because it is done in container not outside. In order to use docker compose you need to have access to all the docker compose yaml files and know how many of those you had - this information is not passed to inside the container.

Reply:
I try, but seem breeze without docker-compose

Reply:
Not inside breeze
<img width="1411" alt="Screen Shot 2020-04-03 at 21 32 37" src="https://user-images.githubusercontent.com/15820530/78366135-a9d40000-75f2-11ea-84d2-c945edca62cb.png">